react/features/base/react/components/native/Container.js

```diff
@@ -58,7 +58,7 @@ export default class Container<P: Props> extends AbstractContainer<P> {
                         accessibilityLabel,
                         accessible,
                         onPress: onClick,
-                        underlayColor
+                        ...touchFeedback && { underlayColor }
```

Comment:
I'm not sure what happens here exactly. What's the effect of `...touchFeedback` ? Is it a string ?

Reply:
This is one of those cases where parentheses help with readability, but they are not needed, so eslint complains. Read it as:

```
...(touchFeedback && { underlayColor })
```
So it will deconstruct the object only if `touchFeedback` truthy.

Reply:
I see, thanks